treesit-fold.el

```diff
@@ -391,14 +398,15 @@ This function is borrowed from `tree-sitter-node-at-point'."
            (end (cdr range))
            (ov (make-overlay beg end))
            (map (make-sparse-keymap)))
-      (keymap-set map "<mouse-1>" #'treesit-fold-open)
+      (define-key map [mouse-1] #'treesit-fold-open)
```

Comment:
Can you explain why the changes here? 🤔

Reply:
It's only a little change that i thought would be a good idea include it, since define-key still used by most packages,
i'm reverting the changes if it's needed

Reply:
No worries! I was just curious about the changes. 👍